example/temperature.cpp

```diff
@@ -0,0 +1,143 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 Yves Delley
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <units/origin.h>
+#include <units/physical/si/base/thermodynamic_temperature.h>
+#include <units/physical/si/prefixes.h>
+#include <units/quantity_io.h>
+#include <units/quantity_point.h>
+#include <units/ratio.h>
+#include <iostream>
+
+using namespace units;
+
+namespace si = units::physical::si;
+
+namespace detail {
+
+struct centikelvin : scaled_unit<ratio(1, 100), si::kelvin> {
```

Comment:
If there is a unit like `centikelvin` than it should be added to `si/base/thermodynamic_themperature.h`. Also `si::prefix` should be set for `si::kelvin`.

Reply:
Please use `prefixed_unit`.

Reply:
Yeah, I used `scaled_unit` because `si::kelvin` was declared with `no_prefix`. I doubt that `centikelvin` is being used, though `millikelvin` definitely is (D-Wave quantum annealers for example operate at that scale). At this point, I just needed an unit that lets me express the offset between the temperatures scales without loss using an integer. I'll update the kelvin.